train_management/views/traintimetabletemplate.py

```diff
@@ -38,3 +39,29 @@ class TrainTimetableTemplateDeleteView(DvzoDeleteView):
     model = TrainTimetableTemplate
     template_name = "train_management/confirm_delete.html"
     success_url = reverse_lazy("train-timetable-template-list")
+
+
+class CreateTimetablesFromTemplateView(DvzoView):
+    permission_required = 'train_management_add_traintimetable'
```

Comment:
Not related to this PR, but is there a possibility to list "all endpoints" vs. "protected endpoints"? This would help manage the permissions!

Reply:
@haustre maybe you know more about that and if it was possible to retrieve that information.